src/test/java/com/adyen/WebhookTest.java

```diff
@@ -425,4 +424,260 @@ public void testManagementWebhookMerchantCreatedParsing() {
+                "      },\n" +
+                "      \"payoutState\": {\n" +
+                "        \"allowPayout\": false,\n" +
+                "        \"payoutLimit\": {\n" +
+                "          \"currency\": \"EUR\",\n" +
+                "          \"value\": 1000\n" +
+                "        },\n" +
+                "        \"disabled\": true,\n" +
+                "        \"disableReason\": \"Reason for disabled status\",\n" +
+                "        \"tierNumber\": 2,\n" +
+                "        \"notAllowedReason\": \"Reason of not allowed\"\n" +
+                "      },\n" +
+                "      \"events\": [\n" +
+                "        {\n" +
+                "          \"event\": \"InactivateAccount\",\n" +
+                "          \"executionDate\": \"2019-01-01T01:00:00+01:00\",\n" +
+                "          \"reason\": \"Reason of event\"\n" +
+                "        }\n" +
+                "      ]\n" +
+                "    },\n" +
+                "    \"description\": \"Description for account holder\",\n" +
+                "    \"legalEntity\": \"Business\",\n" +
+                "    \"primaryCurrency\": \"EUR\",\n" +
+                "    \"verification\": {\n" +
+                "      \"accountHolder\": {\n" +
+                "        \"checks\": [\n" +
+                "          {\n" +
+                "            \"type\": \"IDENTITY_VERIFICATION\",\n" +
+                "            \"status\": \"PENDING\",\n" +
+                "            \"summary\": {\n" +
+                "              \"kycCheckCode\": 100,\n" +
+                "              \"kycCheckDescription\": \"KYC check summary description\"\n" +
+                "            },\n" +
+                "            \"requiredFields\": [\n" +
+                "              \"field.missing\"\n" +
+                "            ]\n" +
+                "          }\n" +
+                "        ]\n" +
+                "      },\n" +
+                "      \"shareholders\": [\n" +
+                "        {\n" +
+                "          \"checks\": [\n" +
+                "            {\n" +
+                "              \"type\": \"IDENTITY_VERIFICATION\",\n" +
+                "              \"status\": \"PENDING\",\n" +
+                "              \"summary\": {\n" +
+                "                \"kycCheckCode\": 100,\n" +
+                "                \"kycCheckDescription\": \"KYC check summary description\"\n" +
+                "              },\n" +
+                "              \"requiredFields\": [\n" +
+                "                \"field.missing\"\n" +
+                "              ]\n" +
+                "            }\n" +
+                "          ],\n" +
+                "          \"shareholderCode\": \"SH000001\"\n" +
+                "        }\n" +
+                "      ],\n" +
+                "      \"bankAccounts\": [\n" +
```

Comment:
Not part of `KYCVerificationResult`
```suggestion
                "      \"bankAccounts\": [\n" +
```

Reply:
example on the specs is wrong, resolving this.